### PR TITLE
api update for pickup request changes

### PR DESCRIPTION
**Pull request by EirikStudsrodOlsen**
Screenshot 2024-01-09 at 10 39 01

### CODE REVIEW

**Comment by WebDevBring** [comment]
[trello.com/c/GRKSuvCH](https://trello.com/c/GRKSuvCH)

**Review by larsandremoen — APPROVED** [review]
LGTM

File: content/api/revision-history/booking/2024-01-08.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Booking API - Changing pickupDetails in api request for Pickup Orders
+publishDate: 2024-01-08
+layout: api
+notanapi: true
+---
+
+We are changing the api request format for in __Pickup Order__ for __pickupDetails__.
```
Comment (tomgag) [inline content/api/revision-history/booking/2024-01-08.md:8]: Can we stick to API (uppercase)?

File: content/api/revision-history/booking/2024-01-08.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Booking API - Changing pickupDetails in api request for Pickup Orders
```
Comment (tomgag) [inline content/api/revision-history/booking/2024-01-08.md:2]: I think maybe we can focus on the fact that there are changes in the Pickup Order API format in the header

File: content/api/revision-history/booking/2024-01-08.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+publishDate: 2024-01-08
+layout: api
+notanapi: true
+---
+
+We are changing the api request format for in __Pickup Order__ for __pickupDetails__.
+New Pickup Order request format uses `package` field, `pallet` field and `postContainer` field to give information on packages, pallets and postContainers.
```
Comment (tomgag) [inline content/api/revision-history/booking/2024-01-08.md:9]: The new ... uses the `package`, `pallet`, and `postContainer` fields to ...

File: content/api/revision-history/booking/2024-01-08.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+Each of these has a `count` field, `weightInGrams` field  and `volumeInDm3` field.
+Check [Order a pickup](/https://developer.bring.com/api/booking/#order-a-pickup-post) for request examples in json/xml.
```
Comment (tomgag) [inline content/api/revision-history/booking/2024-01-08.md:11]: Check -> See

File: content/api/revision-history/booking/2024-01-09.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: Booking API - Changes in Pickup Order API format
+publishDate: 2024-01-09
+layout: api
+notanapi: true
+---
+
+We are changing the API request format for in __Pickup Order__ for __pickupDetails__.
```
Comment (Igor-J86) [inline content/api/revision-history/booking/2024-01-09.md:8]: Would it make more sense putting it like this?
We are changing the API request format for __pickupDetails__ in the  __Pickup Order__ request.

File: content/api/revision-history/booking/2024-01-09.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+The new Pickup Order request format uses the `package`, `pallet` and `postContainer` field to give information on packages, pallets and postContainers.
+Each of these has a `count` field, `weightInGrams` field  and `volumeInDm3` field.
+See [Order a pickup](/https://developer.bring.com/api/booking/#order-a-pickup-post) for request examples in json/xml.
```
Comment (Igor-J86) [inline content/api/revision-history/booking/2024-01-09.md:11]: I think there‘s a "/" before htttps that shouldn‘t be there..?

**Comment by Igor-J86** [comment]
We have also added the possibility to make some important messages appear on the API page itself + on the API updates page in the right column. If you think this message is of such importance that it should stick out for a while, you can also look into setting `isImportant: true` in the markdown file.

Ref: [https://github.com/bring/developer-site/pull/1625](https://github.com/bring/developer-site/pull/1625)

**Pull request merged**